web/app/charts/MapChart.js

```diff
@@ -22,11 +22,14 @@ bluewave.charts.MapChart = function(parent, config) {
             left: 82
         }
     };
-    var svg, mapArea; //d3 elements
+    var svg, mapArea, mapCenter; //d3 elements
```

Comment:
I don't think we need the mapCenter variable anymore so let's remove

Reply:
It will be removed.